app/models/account.rb

```diff
@@ -217,10 +217,12 @@ def unsilence!
   end
 
   def suspended?
-    suspended_at.present?
+    suspended_at.present? && !instance_actor?
   end
 
   def suspend!(date = Time.now.utc)
+    return if instance_actor?
```

Comment:
This should maybe be changed in AccountPolicy as well to give errors through admin API instead of silently not doing anything.